src/System.Memory/ref/System.Memory.cs

```diff
@@ -77,10 +77,7 @@ public static class Span
         public static ReadOnlySpan<byte> AsBytes<T>(this ReadOnlySpan<T> source) where T : struct { throw null; }
         public static Span<TTo> NonPortableCast<TFrom, TTo>(this Span<TFrom> source) where TFrom : struct where TTo : struct { throw null; }
         public static ReadOnlySpan<TTo> NonPortableCast<TFrom, TTo>(this ReadOnlySpan<TFrom> source) where TFrom : struct where TTo : struct { throw null; }
-
-        public static ReadOnlySpan<char> Slice(this string text) { throw null; }
-        public static ReadOnlySpan<char> Slice(this string text, int start) { throw null; }
-        public static ReadOnlySpan<char> Slice(this string text, int start, int length) { throw null; }
+        public static ReadOnlySpan<char> AsSpan(this string text) { throw null; }
```

Comment:
I have asked before ... what's the plan for `Span` vs. `SpanExtensions`? I thought you wanted to move everything to SpanExtensions in the public surface. It looks odd that AsSpan(string) is on one type, but AsSpan(T[]) is on a different type.